src/workers/parser/tests/getSymbols.spec.js

```diff
@@ -25,6 +25,7 @@ cases(
     },
     { name: "component", file: "component" },
     { name: "react component", file: "frameworks/component" },
-    { name: "flow", file: "flow" }
+    { name: "flow", file: "flow" },
+    { name: "jsx", file: "frameworks/jsx" }
```

Comment:
you should run `jest -u src/workers/parser` to see this run.

Also, because each test uses `formatSymbols` we should make sure it shows the boolean